TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -42,6 +42,94 @@
 
 using namespace utest::v1;
 
+#if defined(__NEWLIB_NANO)
+
+#define MAX_STRING_SIZE 100
+//Global buffer used by test cases to store string literal
+char expected_string[MAX_STRING_SIZE];
+
+typedef enum {
+    RADIX_10,//radix base for decimal
```

Comment:
```suggestion
    RADIX_10 = 10,
```
Radix and base mean the same thing, writing `radix base` does not mean anything. I also find the enum value comments unnecessary. If you insist of having (obvious) comments then perhaps use `\\ Decimal system`